<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReaderEditView</class>
 <widget class="QWidget" name="ReaderEditView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>422</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>读者编辑</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
    background-color: #f5f7fa;
    font-family: &quot;Microsoft YaHei&quot;, &quot;Arial&quot;;
    font-size: 14px;
}</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="1">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>75</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="labelReaderNo">
       <property name="text">
        <string>读者编号：*</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditName">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入读者姓名</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="labelPhone">
       <property name="text">
        <string>电话：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="labelName">
       <property name="text">
        <string>姓名：*</string>
       </property>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="labelMaxBorrowLimit">
       <property name="text">
        <string>最大借书限制：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditEmail">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入电子邮箱</string>
       </property>
      </widget>
     </item>
     <item row="8" column="3">
      <widget class="QPushButton" name="BtnReaderOk">
       <property name="styleSheet">
        <string notr="true">QPushButton#BtnReaderOk {
    background-color: #409eff;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 8px 20px;
    font-weight: 500;
}

QPushButton#pushButtonOk:hover {
    background-color: #66b1ff;
}

QPushButton#pushButtonOk:pressed {
    background-color: #3a8ee6;
}</string>
       </property>
       <property name="text">
        <string>确定</string>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="labelIdCard">
       <property name="text">
        <string>身份证号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditReaderNo">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入读者编号</string>
       </property>
      </widget>
     </item>
     <item row="6" column="0" colspan="2">
      <widget class="QLabel" name="labelCurrentBorrowed">
       <property name="text">
        <string>当前借书数量：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="labelEmail">
       <property name="text">
        <string>邮箱：</string>
       </property>
      </widget>
     </item>
     <item row="7" column="1" colspan="3">
      <widget class="QComboBox" name="comboBoxStatus">
       <item>
        <property name="text">
         <string>正常</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>冻结</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>挂失</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="4" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditIdCard">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入身份证号码</string>
       </property>
      </widget>
     </item>
     <item row="5" column="2" colspan="2">
      <widget class="QSpinBox" name="spinBoxMaxBorrowLimit">
       <property name="suffix">
        <string>本</string>
       </property>
       <property name="value">
        <number>5</number>
       </property>
      </widget>
     </item>
     <item row="8" column="1" colspan="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Orientation::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="8" column="0">
      <widget class="QPushButton" name="BtnReaderCancel">
       <property name="styleSheet">
        <string notr="true">QPushButton#BtnReaderCancel {
    background-color: #909399;
    color: white;
    border: none;
    border-radius: 4px;
    padding: 8px 20px;
    font-weight: 500;
}

QPushButton#pushButtonCancel:hover {
    background-color: #a6a9ad;
}

QPushButton#pushButtonCancel:pressed {
    background-color: #82848a;
}</string>
       </property>
       <property name="text">
        <string>取消</string>
       </property>
      </widget>
     </item>
     <item row="6" column="2" colspan="2">
      <widget class="QSpinBox" name="spinBoxCurrentBorrowed">
       <property name="readOnly">
        <bool>true</bool>
       </property>
       <property name="suffix">
        <string>本</string>
       </property>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QLabel" name="labelStatus">
       <property name="text">
        <string>状态：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1" colspan="3">
      <widget class="QLineEdit" name="lineEditPhone">
       <property name="styleSheet">
        <string notr="true">QLineEdit, QComboBox {
    padding: 8px;
    border: 1px solid #dcdfe6;
    border-radius: 4px;
    background-color: white;
    min-height: 20px;
}

QLineEdit:focus, QComboBox:focus {
    border-color: #409eff;
    outline: none;
}

QComboBox::drop-down {
    border: none;
    width: 20px;
}

QComboBox::down-arrow {
    image: none;
    border-left: 4px solid transparent;
    border-right: 4px solid transparent;
    border-top: 6px solid #666;
    width: 12px;
    height: 12px;
}</string>
       </property>
       <property name="placeholderText">
        <string>请输入联系电话</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="1" column="2">
    <spacer name="horizontalSpacer_3">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>75</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Orientation::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>35</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
